src/js/pages/system/UnitsHealthDetail.js

```diff
@@ -47,6 +55,24 @@ class UnitsHealthDetail extends mixin(StoreMixin) {
     UnitHealthStore.fetchUnitNodes(this.props.params.unitID);
   }
 
+  onUnitHealthStoreUnitSuccess() {
+    let isLoading = this.state.isLoading & ~FLAGS.UNIT_LOADED;
```

Comment:
I'm up for a different solution, but what I don't want is to have `n` number of variables tracking if different things have loaded. I think this is the right tool for the job, and I'm hoping you guys see it once you get used to looking at it. 
Maybe this example helps:
```
let myVar = 0b111;
let MY_FLAG = 0b11;

myVar &= ~MY_FLAG;
myVar.toString(2); // 100
myVar &= ~MY_FLAG;
myVar.toString(2); // 100 (it doesn't keep flipping bits)
```

It's also really nice to do `if (this.state.isLoading)` in the `render` method. Rather than `if (fooIsLoading && barIsLoading && ...)`.